plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -53,6 +53,10 @@ func eventsRule() rbacv1.PolicyRule {
 	return rbacv1helpers.NewRule("create", "update", "patch").Groups(legacyGroup).Resources("events").RuleOrDie()
```

Comment:
it would make more sense to add eventsGroup here... there's no case I can think of where we would want to authorize one and not the other

Reply:
@liggitt - for now only kube-scheduler will use `events.k8s.io`. Once we switch the other components we can refactor this

Reply:
it doesn't matter... it is less error-prone to update the method all components are using to gain event rules than to do it piecemill

Reply:
sgtm